crates/ruff_python_parser/src/snapshots/ruff_python_parser__string__tests__parse_fstring_self_doc_trailing_space.snap

```diff
@@ -31,7 +13,13 @@ expression: parse_ast
                     ctx: Load,
                 },
             ),
-            conversion: Repr,
```

Comment:
note that we are removing the `Repr` setting here (implied by the self-doc)

Reply:
Could you explain your reasoning for removing it in more detail?

Reply:
now that we are explicitly modelling self-documenting formatted values we can use `conversion` for _explicit conversions only (this one was implicit)

Reply:
I see, this seems to be now consistent with non-debug strings. 

My personal preference would be to remove `Conversion::None` altogether because (please correct me if I'm wrong) `Conversion::Repr` is the default if no conversion is specified. Meaning, lint rules that e.g. should capture all usages of expressions using `__repr__`  shouldn't have to test for both `None` and `Repr`. But this change is unrelated to this PR.

Reply:
i guess we could change it for and `Option<ConversionFlag>` but surely we want to track any flags set in the source?

Reply:
`Option<ConversionFlag>` is similar to what we have today:

* `Conversion::None` -> `None`
* `Conversion::Repr` -> `Some(ConversionFlags::Repr)`
* `Conversion::Str` -> `Some(ConversionFlags::Str)`

> i guess we could change it for an Option<ConversionFlag> but surely we want to track any flags set in the source?

Yeah, let's keep it the way it is. Knowing the conversion specified in the source would be helpful if we e.g. want to lint for unnecessary conversion specifiers. 

We could add a method on `FormattedValue` that returns the *used* conversion at runtime which always returns a none `None` value.